watson/watson.py

```diff
@@ -288,9 +291,14 @@ def stop(self, stop_at=None):
         if stop_at > arrow.now():
             raise WatsonError('Task cannot end in the future.')
 
+        if note is None:
+            note = old.get('note')
```

Comment:
So that means that if a not is passed with `watson stop`, it will override a previous note passed via `watson start`? Shouldn't we try to concatenate them somehow, or at least not silently drop it.

Reply:
I’m not sure what people would prefer: concatenating or dropping. Either way silently dropping is bad I agree.

How about printing a message that the old note from start is being overwritten, print it, the print the new one?  The user can edit the frame if they want to amend the note. It will be obvious and they will have a copy of the old note, if needed. 

My hypothesis is overwriting will be more common. When you start a task you might have a general idea of what to do but not the full picture. By the end of the task you can describe exactly what happened.

Reply:
Ok for a warning + dropping it :+1: . I guess people will either pass a message to start or stop, but not both.